blazor/document-editor/getting-started/web-app.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Getting Started with Blazor DocumentEditor in Web App | Syncfusion
-description: Checkout and learn about the documentation for getting started with Blazor DocumentEditor Component in Blazor Web App.
+description: Checkout and learn about the documentation for getting started with Blazor DocumentEditor component, its elements and more.
```

Comment:
Checkout and learn about the documentation for getting started with Blazor Document Editor component, its elements, and more.

Reply:
resolved